### PR TITLE
feat: hertz binding refactor

### PR DESCRIPTION
**Pull request by FGYFFFF**
#### What type of PR is this?

feat
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.
hertz 绑定重构文档

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en: hertz binding refactor doc
zh(optional):  hertz 绑定重构文档

### CODE REVIEW

**Comment by li-jin-gou** [comment]
PATL @Skyenought  @chaoranz758

File: content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md [hunk 1]
```diff
@@ -41,6 +41,22 @@ func main() {
...
+|:--------------------|:----------------------------------------------------------------------------------------------------------------------------------------|
+| ctx.BindAndValidate | 利用下述的 go-tag 进行参数绑定，并在绑定成功后做一次参数校验(如果有校验 tag 的话)                                                                                        |
+| ctx.Bind                | 同 "BindAndValidate" 但是不做参数校验                                                                                                            |
+| ctx.BindQuery           | 绑定所有 Query 参数，相当于给每一个 field 声明一个 'query' tag，适用于没写 tag 的场景                                                                              |
+| ctx.BindHeader          | 绑定所有 Header 参数，相当于给每一个 field 声明一个 'header' tag，适用于没写 tag 的场景                                                                            |
+| ctx.BindPath            | 绑定所有 Path 参数，相当于给每一个 field 声明一个 'path' tag，适用于没写 tag 的场景                                                                                |
+| ctx.BindForm            | 绑定所有 Form 参数，相当于给每一个 field 声明一个 'form' tag，需要 Content-Type 为: "application/x-www-form-urlencoded"/"multipart/form-data",  适用于没写 tag 的场景 |
```
Comment (chaoranz758) [inline content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md:54]: Content-Type 使用代码块形式，下同

File: content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md [hunk 2]
```diff
@@ -194,8 +434,25 @@ func init() {
 
 ### 配置 looseZero
 
-在一些场景下，前端有时候传来的信息只有 key 没有 value，这会导致绑定数值类型的时候，会报错 `cause=parameter type does not match binding data`。
-这时需要配置 looseZero 模式（[demo](https://github.com/cloudwego/hertz-examples/tree/main/binding/loose_zero) ），使用方法如下：
+在一些场景下，前端有时候传来的信息只有 key 没有 value，这会导致绑定数值类型的时候报错 这时需要配置 looseZero 模式，使用方法如下：
```
Comment (chaoranz758) [inline content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md:437]: 缺少逗号

File: content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md (hunk 1 above)
Comment (FGYFFFF) [reply content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md]: done

File: content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md (hunk 2 above)
Comment (FGYFFFF) [reply content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md]: done

**Review by Duslia — APPROVED** [review]

**Review by welkeyever — APPROVED** [review]

**Comment by li-jin-gou** [comment]
记得 + 英文 @FGYFFFF

**Pull request merged**